### PR TITLE
S3 populate dropdown

### PR DESCRIPTION
**Pull request by imogenhallett**
Created select dropdown for event type, populated by options from database.

Files to be reviewed:
src/Controllers/AddEventPageController.php
src/Factories/AddEventPageControllerFactory.php
src/Factories/EventModelFactory.php
src/Models/EventModel.php
src/ViewHelpers/AddEventPageViewHelper.php

tests/Controllers/AddEventPageControllerTest.php
tests/Factories/AddEventPageControllerFactoryTest.php
tests/Factories/EventModelFactoryTest.php
tests/Models/EventModelTest.php
tests/ViewHelpers/AddEventPageViewHelperTest.php

### CODE REVIEW

File: src/Classes/Factories/AddEventPageControllerFactory.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+use Portal\Controllers\AddEventPageController;
+
+class AddEventPageControllerFactory
+{
+    /**
+     * @param $c
+     * @return AddEventPageController
+     */
+    public function __invoke($c)
```
Comment (hoptond) [inline src/Classes/Factories/AddEventPageControllerFactory.php]: [birds/plane/etc comment from previous review]

File: src/Classes/Factories/EventModelFactory.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+use Portal\Models\EventModel;
+
+class EventModelFactory
+{
+    /**
+     * @param $c
+     * @return EventModel
+     */
+    public function __invoke($c)
```
Comment (hoptond) [inline src/Classes/Factories/EventModelFactory.php]: see above

File: src/Classes/Factories/AddEventPageControllerFactory.php (hunk 1 above)
Comment (imogenhallett) [reply src/Classes/Factories/AddEventPageControllerFactory.php]: Renamed in 114d7f3

File: src/Classes/Factories/EventModelFactory.php (hunk 2 above)
Comment (imogenhallett) [reply src/Classes/Factories/EventModelFactory.php]: Renamed in 114d7f3

**Pull request merged**